frontend/test/__support__/ui.tsx

```diff
@@ -303,10 +303,10 @@ export function getBrokenUpTextMatcher(textToFind: string): MatcherFunction {
  * @see https://github.com/metabase/metabase/pull/34272#discussion_r1342527087
  * @see https://metaboat.slack.com/archives/C505ZNNH4/p1684753502335459?thread_ts=1684751522.480859&cid=C505ZNNH4
  */
-export const waitForLoaderToBeRemoved = async () => {
+export const waitForLoaderToBeRemoved = async (options?: waitForOptions) => {
```

Comment:
needed this to debug something, but it's kind of nice to keep. happy to remove if folks have opinions.